### DISCUSSION TITLE
Is this mod abandoned

### DISCUSSION THREAD

**Discussion by ImagineUsingHammer** [BEEmod/BEE2-items#4213] [Tech Support / Q&A] [answered]
Not an issue, you can close it

But I'm just curious because this mod wasn't updated a very long time.

Or the developers are working on a fully new and better bee 5?

**Comment by LautaroL20** [comment, marked as answer]
This mod is pretty much alive, TeamSpen210 has even release a closed beta in the Discord server in order to get ready a BEE2.4 release soon.

**Reply by ItzAlmaZ** [reply]
Can i get invite link in discord server?

**Reply by BenVlodgi** [reply]
[Additional Information](https://github.com/BEEmod/BEE2-items#additional-information)
